build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import io.spring.gradle.dependencymanagement.internal.dsl.StandardDependencyManagementExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "3.1.4" apply false
+    id("io.spring.dependency-management") version "1.1.3" apply false
+    kotlin("jvm") version "1.9.10" apply false
+    kotlin("plugin.spring") version "1.9.10" apply false
+    kotlin("plugin.jpa") version "1.9.10" apply false
+    id("com.github.davidmc24.gradle.plugin.avro") version "1.8.0" apply false
```

Comment:
Hoort deze plugin niet enkel in de avro component thuis?

Reply:
klopt, met `apply false` zeggen we dat we het niet toepassen, alleen een versie defineren.

Reply:
Dat snap ik, mijn onderliggende issue was hierbij dat, als we een service gaan implementeren zonder gebruik te maken van Avro, we op meerdere plekken code moeten verwijderen...
(Is waarschijnlijk niet onder uit te komen, als je versiebeheer op een plek wil hebben...)